tools/plugins/com.liferay.ide.hook.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -3,5 +3,5 @@ bundle.name = Liferay IDE Hook UI
 bundle.vendor = Liferay, Inc.
 category.name = Liferay
 hook.editor.name = Liferay Hook Editor
-hook.wizard.name = Liferay Hook Configuration
+hook.wizard.name = Liferay Hook Configuration (Deprecated)
```

Comment:
hey @jtydhr88  I'm worried about the word 'deprecated'.  I'm not sure we want to list that in the UI.  Lets discuss this on a call next week.